deep-learning/_posts/2017-03-31-densenet.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: review
+title:  "DenseNet : Densely Connected Convolutional Networks"
+tags:   deep-learning CNN classification
+author: Carl Lemaire
+pdf:    https://arxiv.org/pdf/1608.06993.pdf
+cite:
+  authors: "Gao Huang, Zhuang Liu, Kilian Q Weinberger, Laurens van der Maaten"
+  title:   "Densely connected convolutional networks"
+  venue:   "arXiv preprint arXiv:1608.06993"
+---
+
+DenseNet is a CNN architecture that includes _dense blocks_. In a dense block, each layer has access to the feature maps of all preceding layers. Figure 1 shows a dense block.
+
+![](/deep-learning/images/densenet/fig1.png)
+
+DenseNets have many advantages :
+
+* They require substantially fewer parameters and less computation to achieve state-of-the-art performances ;
+* They naturally integrate the properties of identity mappings, deep
+supervision, and diversified depth ;
```

Comment:
Not sure I understand what identity mapping, deep supervision and diversified depth means...  You may refer to a section in the paper.